### ISSUE TITLE
Is binary file download available?

### ISSUE THREAD

**Issue by glyslxq** [RediSearch/RSCoordinator#338]
emmm...

I encountered a problem compiling

**Comment by glyslxq** [comment]
[  1%] Built target coordinator-core
[  1%] Built target metaphone
[ 25%] Built target rscore
[ 26%] Built target rmutil
[ 27%] Built target friso
[ 37%] Built target snowball
[ 61%] Built target redisearch-oss
[ 62%] Built target redismock
[ 63%] Building C object src/dep/rmr/CMakeFiles/rmr.dir/conn.c.o
/root/RSCoordinator/src/dep/rmr/conn.c: In function ‘MRConn_Connect’:
/root/RSCoordinator/src/dep/rmr/conn.c:445:38: error: ‘REDIS_OPT_NOAUTOFREEREPLIES’ undeclared (first use in this function)
                           .options = REDIS_OPT_NOAUTOFREEREPLIES,
                                      ^
/root/RSCoordinator/src/dep/rmr/conn.c:445:38: note: each undeclared identifier is reported only once for each function it appears in
make[2]: *** [src/dep/rmr/CMakeFiles/rmr.dir/conn.c.o] Error 1
make[1]: *** [src/dep/rmr/CMakeFiles/rmr.dir/all] Error 2
make: *** [all] Error 2

**Comment by xiaohei31** [comment]
RSCoordinator is deprecated for RediSearch>=2.2.
RSCoordinator is already merged into RediSearch.
Please try make RediSearch by make build COORD=1
